.github/workflows/node.js.yml

```diff
@@ -1,15 +1,26 @@
-name: Reactibot deployment
+name: Node.js CI
 
-on:
-  push:
-    branches:
-      - master
-      - develop
-      - feature/actions
+on: [push, pull_request]
 
 jobs:
   build:
     runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install
+        run: npm ci
+      - name: Test
+        run: npm test
+      - name: Build
+        run: npm run build --if-present
+
+  deployment:
+    needs: build
+    if: github.ref == 'master' || github.ref == 'develop' || github.ref == 'feature/actions'
```

Comment:
Do we have a `feature/actions` branch? or is this something setup for strictly testing our CI?

Reply:
It was for testing the branch that implemented the first action without having to push directly to master/dev because we're not degenerates

Reply:
That was from the old deployment workflow. I'm assuming the purpose is to be able to deploy updates to the workflow without merging into master. It also let me test this PR out, and I finally got the deploy working!